templates/supdfgen/partials/fellesInfo.hbs

```diff
@@ -1,21 +1,21 @@
 <div class="info">
     <h3>Du har rett til å klage</h3>
     <p>
-        Du kan klage innen seks uker fra den datoen du mottok vedtaket. Du finner skjema og informasjon på nav.no/klage.
+        Du kan klage innen seks uker fra den datoen du mottok vedtaket. Du finner skjema og informasjon på <a>www.nav.no/klage</a>.
```

Comment:
Denne blir redirected til: `https://klage.nav.no/nb/`. Jeg antar `www.nav.no/klage` føles tryggere og er mer forståelige for den gjengse bruker?

Burde vi legge på https i tillegg til eller istedenfor www?

Reply:
@hestad Tenker den egentlig burde stått som `nav.no/klage` kanskje, men da funket ikke lenke ut av boksen med a-taggen uten å sette på href i tillegg. 

https:// blir kanskje litt meget i papir-brevene, men et alternativ er å ha en kort-versjon (nav.no/xxx) i teksten med en href til full URL (https://www.nav.no/xxx). Blir bare litt mer å administrere. Hva tenker du?

Reply:
Den siste varianten høres mer forlokkende ut, men det er mer jobb som du sier. Vi kan godt merge det som det er.